packages/components/src/components/segmented-control/segmented-control-item/segmented-control-item-utils.ts

```diff
@@ -12,3 +12,7 @@ export const getButtonAttributes = (isSelected: boolean, isDisabled: boolean): A
   ...getButtonBaseAriaAttributes(isDisabled, false),
   ...parseAndGetAriaAttributes({ 'aria-pressed': isSelected }),
 });
+
+export const getIconColor = (isDisabled: boolean) => {
```

Comment:
Explicit return type is missing (and probably produces a lint warning).

Reply:
fixed it